process/logsevents/fungibleESDTProcessor.go

```diff
@@ -94,6 +94,8 @@ func (fep *fungibleESDTProcessor) processEventDestination(args *argsProcessEvent
 	}
 
 	if senderShardID != receiverShardID {
+		encodedSender := fep.pubKeyConverter.Encode(args.event.GetAddress())
+		args.pendingBalances.addInfo(encodedSender, tokenID, 0, big.NewInt(0).String())
 		args.pendingBalances.addInfo(encodedAddr, tokenID, 0, big.NewInt(0).String())
```

Comment:
might rename this to `encodedReceiver` instead of `encodedAddr`

Reply:
done.